docs/_data/mesheryctlcommands/meshes/deploy/command/deploy.yaml

```diff
@@ -0,0 +1,9 @@
+name: deploy
+description: Deploy a service mesh into the cluster
+usage: |
+
+    mesheryctl mesh deploy [flags]
+flags:
+  - name: --adapter
+  - name: --tokenpath
+  - name: --namespace
```

Comment:
🤔 this seems good, including flags... Let's see what lee says on this...

Reply:
yes, this has the same kind of flags as **mesh validate** is having except **namespace**. You can have a look. 

It looks fine to me too.

// @leecalcote

Reply:
You're not leaving me a lot of room to complain here, gents.

This is looking fairly solid to me.